packages/patternfly-4/react-core/src/layouts/Page/Page.js

```diff
@@ -16,25 +17,73 @@ const propTypes = {
   /** Header component (e.g. <PageHeader />) */
   header: PropTypes.node,
   /** Sidebar component for a side nav (e.g. <PageSidebar />) */
-  sidebar: PropTypes.node
+  sidebar: PropTypes.node,
+  /** condensed header on scroll */
+  useCondensed: PropTypes.bool,
```

Comment:
NIT: Probably should capitalize these comments since they get generated to our docs.